### PR TITLE
pkp/pkp-lib#1046 Add ability to save submission step 3 without submitting document

### PR DESCRIPTION
**Pull request by asmecher**

### CODE REVIEW

File: classes/submission/form/SubmissionSubmitStep4Form.inc.php [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * @class SubmissionSubmitStep4Form
+ * @ingroup submission_form
+ *
+ * @brief Form for Step 4 of author submission.
+ */
+
+import('lib.pkp.classes.submission.form.PKPSubmissionSubmitStep4Form');
+import('classes.submission.SubmissionMetadataFormImplementation');
```
Comment (beghelli) [inline classes/submission/form/SubmissionSubmitStep4Form.inc.php]: is this required here?

File: pages/submission/SubmissionHandler.inc.php [hunk 2]
```diff
@@ -54,12 +54,13 @@ function fetchChoices($args, $request) {
...
 	 */
-	protected function _getStepsNumberAndLocaleKeys() {
+	function getStepsNumberAndLocaleKeys() {
 		return array(
 			1 => 'submission.submit.prepare',
 			2 => 'submission.submit.upload',
 			3 => 'submission.submit.catalog',
-			4 => 'submission.submit.nextSteps'
+			4 => 'submission.submit.confirmation',
+			5 => 'submission.submit.nextSteps',
```
Comment (beghelli) [inline pages/submission/SubmissionHandler.inc.php]: not doing this for OJS? If yes, why not leaving this in library?

File: classes/submission/form/SubmissionSubmitStep4Form.inc.php (hunk 1 above)
Comment (asmecher) [inline classes/submission/form/SubmissionSubmitStep4Form.inc.php]: Nope, will remove.

File: pages/submission/SubmissionHandler.inc.php (hunk 2 above)
Comment (asmecher) [inline pages/submission/SubmissionHandler.inc.php]: Currently the locale keys for OMP and OJS differ here. I've opted not to reconcile that yet; will have a look when I port over to OJS.

**Pull request merged**